common/lib/capa/capa/templates/status_icon.html

```diff
@@ -0,0 +1,10 @@
+<%page expression_filter="h" args="status"/>
+<%! from django.utils.translation import ugettext as _ %>
+
+% if status == 'correct':
+  <span class="status-icon ${status.classname} fa fa-check"></span>
```

Comment:
add aria-hidden in these new spans
I think you can skip Font Awesome class, so that for the unsubmitted state you don't have to remove the status icon span. So `.unsubmitted .status-icon {content: '''}`